internal/tui/bubbles/git/types/utils.go

```diff
@@ -0,0 +1,10 @@
+package types
+
+import "github.com/muesli/reflow/truncate"
+
+func TruncateString(s string, max int, tail string) string {
+	if max < 0 {
+		max = 0
+	}
+	return truncate.StringWithTail(s, uint(max), tail)
```

Comment:
maybe append a `...` to indicate it was truncated?

Reply:
Yes, I'm appending `...` in places where this function is called like https://github.com/charmbracelet/soft-serve/pull/77/files#diff-f205e7d31efe3ad5e269cdb1b63b7753e7c6b244ab2c73feedd8abb3d417c1c3R99